### PR TITLE
fix: features query

### PR DESCRIPTION
**Pull request by fgerschwiler**

### CODE REVIEW

**Review by livio-a — DISMISSED** [review]

**Review by adlerhurst — CHANGES_REQUESTED** [review]

File: internal/query/projection/feature_test.go [hunk 1]
```diff
@@ -97,6 +96,36 @@ func TestFeatureProjection_reduces(t *testing.T) {
...
+				projection:       FeatureTable,
+				executer: &testExecuter{
+					executions: []execution{
+						{
+							expectedStmt: "UPSERT INTO zitadel.projections.features (aggregate_id, change_date, sequence, is_default) VALUES ($1, $2, $3, $4)",
+							expectedArgs: []interface{}{
+								"agg-id",
+								anyArg{},
+								uint64(15),
+								false,
```
[the remark this answers is not in the document]
Comment (livio-a) [reply internal/query/projection/feature_test.go:122]: yes, but reduceFeatureSet will always set it

File: internal/authz/repository/eventsourcing/eventstore/token_verifier.go [hunk 2]
```diff
@@ -110,7 +111,7 @@ func (repo *TokenVerifierRepo) ProjectIDAndOriginsByClientID(ctx context.Context
 }
 
 func (repo *TokenVerifierRepo) CheckOrgFeatures(ctx context.Context, orgID string, requiredFeatures ...string) error {
-	features, err := repo.View.FeaturesByAggregateID(orgID)
+	features, err := repo.Query.FeatureByOrgID(ctx, orgID)
```
[the remark this answers is not in the document]
Comment (livio-a) [reply internal/authz/repository/eventsourcing/eventstore/token_verifier.go:114]: oh yes, you're right

File: internal/authz/repository/eventsourcing/eventstore/token_verifier.go [hunk 3]
```diff
@@ -280,28 +281,11 @@ func (u *TokenVerifierRepo) getIAMByID(ctx context.Context) (*iam_model.IAM, err
...
-	if esErr != nil {
-		logging.Log("EVENT-PSoc3").WithError(esErr).Debug("error retrieving new events")
-		return esErr
-	}
-	featuresCopy := *features
-	for _, event := range events {
-		if err := featuresCopy.AppendEvent(event); err != nil {
-			return checkFeatures(features, requiredFeatures...)
-		}
+	features, err := repo.Query.DefaultFeature(ctx)
```
[the remark this answers is not in the document]
Comment (livio-a) [reply internal/authz/repository/eventsourcing/eventstore/token_verifier.go:284]: i'll add all the missing `s`

File: internal/query/features_test.go [hunk 4]
```diff
@@ -0,0 +1,348 @@
...
+					[]driver.Value{
+						"aggregate-id",
+						testNow,
+						uint64(20211115),
+						true,
+						"tier-name",
+						"tier-description",
+						1,
+						"state-description",
+						604800000000000,
```
Comment (adlerhurst) [inline internal/query/features_test.go:137]:
```suggestion
						7 * 24 * time.Hour,
```

File: internal/query/features_test.go [hunk 5]
```diff
@@ -0,0 +1,348 @@
...
+					[]driver.Value{
+						"aggregate-id",
+						testNow,
+						uint64(20211115),
+						true,
+						nil,
+						nil,
+						1,
+						nil,
+						604800000000000,
```
Comment (adlerhurst) [inline internal/query/features_test.go:252]:
```suggestion
						7 * 24 * time.Hour,
```

**Review by adlerhurst — APPROVED** [review]

**Pull request merged**